src/promql.term.js

```diff
@@ -0,0 +1,88 @@
+export const promqlSyntax = {
+    "MatchOp": [
+        "=",
+        "!=",
+        "=~",
+        "!~"
+    ],
+    "BinaryExpr": [
```

Comment:
These are technically not binary expressions, but binary operators, so `BinaryOp` would be a better name.

Reply:
actually the name here is following the NodeName you put in the grammar:

```
BinaryExpr {
  Expr !pow BinOp<"^">    BinModifier Expr |
  Expr !mul BinOp<"*">    BinModifier Expr |
  Expr !mul BinOp<"/">    BinModifier Expr |
  Expr !mul BinOp<"%">    BinModifier Expr |
  Expr !add BinOp<"+">    BinModifier Expr |
  Expr !add BinOp<"-">    BinModifier Expr |
  Expr !eql BinOp<"==">   BinModifier Expr |
  Expr !eql BinOp<">=">   BinModifier Expr |
  Expr !eql BinOp<">">    BinModifier Expr |
  Expr !eql BinOp<"<">    BinModifier Expr |
  Expr !eql BinOp<"<=">   BinModifier Expr |
  Expr !eql BinOp<"!=">   BinModifier Expr |
  Expr !and ckw<"and">    BinModifier Expr |
  Expr !and ckw<"unless"> BinModifier Expr |
  Expr !or  ckw<"or">     BinModifier Expr
}
```

From my point of view it makes more sense to put `BinaryExpr` as it is the top node that handles every kind of BinOp or "and" / "unless" / "or".
But I can change it if you really think it makes more sense to put `BinaryOp`

Reply:
I changed it.